src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -59,28 +60,32 @@ public class ConverterRegistry {
 
 	private static final Map<ModElementType<?>, List<IConverter>> converters = new HashMap<>() {{
 		put(ModElementType.ADVANCEMENT, List.of(new AchievementFixer(), new AdvancementTextureConverter()));
-		put(ModElementType.ARMOR, List.of(new ArmorTexturesConverter()));
+		put(ModElementType.ARMOR, List.of(new ArmorTexturesConverter(), new SpecialInformationConverter()));
 		put(ModElementType.BIOME, List.of(new BiomeSpawnListConverter(), new BiomeDefaultFeaturesConverter(),
 				new BiomeFrozenTopLayerConverter(), new BiomeGenParametersConverter(),
 				new BiomeCustomFeaturesConverter()));
 		put(ModElementType.BLOCK,
 				List.of(new BlockLuminanceFixer(), new BlockBoundingBoxFixer(), new BlockLightOpacityFixer(),
 						new BlockRequiresCorrectToolConverter(), new BlockOreReplacementBlocksFixer(),
-						new BlockGenerationConditionRemover()));
-		put(ModElementType.DIMENSION, List.of(new DimensionLuminanceFixer()));
-		put(ModElementType.FLUID, List.of(new FluidNameFixer(), new FluidGenToFeatureConverter()));
+						new BlockGenerationConditionRemover(), new SpecialInformationConverter()));
+		put(ModElementType.DIMENSION, List.of(new DimensionLuminanceFixer(), new SpecialInformationConverter()));
+		put(ModElementType.FLUID,
+				List.of(new FluidNameFixer(), new FluidGenToFeatureConverter(), new SpecialInformationConverter()));
 		put(ModElementType.COMMAND, List.of(new CommandParameterBlockFixer()));
 		put(ModElementType.GAMERULE, List.of(new GameruleDisplayNameFixer(), new BooleanGameRulesConverter()));
 		put(ModElementType.GUI,
 				List.of(new GUIBindingInverter(), new GUICoordinateConverter(), new GUITexturesConverter(),
 						new GUIComponentNamer(), new SlotInteractionsConverter()));
+		put(ModElementType.ITEM,

```

Comment:
Please only add converter here, not change order and now item is here but was below before and this makes the file hard to review + can introduce mistakes

Reply:
Done!